### PR TITLE
First version of the 1DPAMYT model and setup.py mods

### PR DESCRIPTION
**Pull request by Gregg140**
## Description

Initial version of a 1DPAMYT model which can be used if the 1DPAMZT sensor fails.

## Interface impacts

The 1DPAMYT model will have a calling sequence similar to 1DPAMZT

### Unit tests

- [ ] No unit tests
- [ ] Mac
- [ x] Linux
- [ ] Windows

### Functional tests

Run with several past load weeks.  Validation plots provided insight as to the goodness of the fit.

### CODE REVIEW

**Review by jazan12 — APPROVED** [review]
Looked good to me

**Comment by jzuhone** [comment]
@Gregg140 can I push up a quick fix to this branch to fix the test failure? It's just a reformatting fix.

File: setup.py [hunk 1]
```diff
@@ -28,9 +29,9 @@
     packages=["acis_thermal_check"],
     use_scm_version=True,
     setup_requires=["setuptools_scm", "setuptools_scm_git_archive"],
-    description="ACIS Thermal Model Library",
-    author="John ZuHone",
-    author_email="john.zuhone@cfa.harvard.edu",
+    description="ACIS Thermal Model Library Program for 1DPAMYT",
+    author="Gregg Germain",
```
[the remark this answers is not in the document]
Comment (jzuhone) [reply setup.py:33]: and `description="ACIS Thermal Model Library"` should stay the same.

**Review by uufrancis — APPROVED** [review]

**Comment by taldcroft** [comment]
@Gregg140 - did you consider factoring out the common code into a base class to adhere to the DRY principle? The only functional difference between your new module and the existing one is changing the MSID name, so that MSID could be a base class attribute.

**Comment by Gregg140** [comment]
> @Gregg140 - did you consider factoring out the common code into a base class to adhere to the DRY principle? The only functional difference between your new module and the existing one is changing the MSID name, so that MSID could be a base class attribute.

 John Z and I discussed this at length yesterday.  John has  identified a few places where we should consider a bigger refactor overall.  And we agreed that we should put this present version in place for now so that we have it and can start running it sooner rather than later.

**Pull request merged**